### ISSUE TITLE
Animation toggle button has limited support for High Contrast Themes (HCM)

### ISSUE THREAD

**Issue by maureenlholland** [mozilla/bedrock#15763]
## Description

[See Bugzilla for details](https://bugzilla.mozilla.org/show_bug.cgi?id=1936865)

> When a High contrast theme (HCM / forced-colors) is active on Windows, the Animation toggle is hard to discern - it does use ButtonFace system color for the background but is missing a border and the label and border are using a color that makes it hard to discern from the page background

NOTE: this is most noticeable on smaller screens/with zoom when the button is icon-only

---

#### Success Criteria
- [ ] ensure the appropriate color pairs are used for forced-colors [media query (HCM on Win)](https://firefox-source-docs.mozilla.org/accessible/HCMMediaQueries.html): ButtonFace with ButtonText for the border/label text and for the control background. Focus indication is expected to use CanvasText system color
